deployment/helm/synopsys-alert/values.yaml

```diff
@@ -41,6 +41,29 @@ cfssl:
   securityContext: {}
   podSecurityContext: {}
 
+# alertdb - configurations for the alertdb Pod
+alertdb:
```

Comment:
Instead of alertdb, can we have it as postgres?

Reply:
Is there a problem with having this as alertdb? Is the name change just a preference?

Reply:
@jamesrichard91 just to match with the container names, so that customers won't get confused.